internal/topo/planner/planner.go

```diff
@@ -463,6 +465,14 @@ func createLogicalPlan(stmt *ast.SelectStatement, opt *api.RuleOption, store kv.
 		p.SetChildren(children)
 		children = []LogicalPlan{p}
 	}
+	windowFuncFields, windowFuncsNames := extractWindowFuncFields(stmt)
```

Comment:
Do we support to use window function in order by? Looks like not supported in the planner.

Reply:
we will support it later.